code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -29,9 +29,8 @@
 /datum/eldritch_knowledge/New()
 	. = ..()
 	var/list/temp_list
-	for(var/X in required_atoms)
-		var/atom/A = X
-		temp_list += list(typesof(A))
+	for(var/atom/required_atom in required_atoms)
```

Comment:
Unless this is supposed to be a filtered list, keeping the same behavior would be this
```suggestion
	for(var/atom/required_atom as anything in required_atoms)
```

Reply:
@ninjanomnom Remember to draft when you make code review, please.